lib/proxy.js

```diff
@@ -0,0 +1,44 @@
+const httpProxy = require("http-proxy");
+
+/**
+ * Simple Http Proxy, proxies request where url is contained in config to target from config
+ *
+ * @param req
+ * @param res
+ * @param {object} configs - proxyConfig from config file
+ * @param {boolean} useHttps - will determine protocol to use
+ * @return {boolean} - will return whether request is being proxied
+ */
+function proxyHandler(req, res, configs, useHttps) {
+    if (!configs || configs.length === 0) {
```

Comment:
after this merged, I will create a config validation function and move this code in there